UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Services/ProductDataServiceTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using FakeItEasy;
+using FluentValidation.Results;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.Services;
+using UKHO.ExchangeSetService.API.Validation;
+using UKHO.ExchangeSetService.Common.Models.Request;
+using UKHO.ExchangeSetService.Common.Models.Response;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Services
```

Comment:
The brackets on the NUnit tags are not required, e,g, [TestFixture], [Test]

Reply:
Hi Greg, although they are not necessarily required they still are legitimate code, I am not going to block a pull request for it

Reply:
We have removed [TestFixture] and [Test] brackets.